app/main.py

```diff
@@ -1 +1,13 @@
 # write your code here
+def copy_file(command: str) -> None:
+    split_command = command.split(" ")
+    if len(split_command) < 3 or split_command[1] == split_command[2]:
```

Comment:
You can use unpacking instead of accessing list items by indexes. For example: a, b = [1,2]

Reply:
Add a check for command name, ensure that it equals exactly to "cp"